### PR TITLE
PCC01 dylannakahodo

### PR DESCRIPTION
**Pull request by dylannakahodo**
Difficulty level (1-10): [1]
Estimated time spent (hours): [1]
Completed (yes/no): [Yes]
I stretched my coding skills (if yes what did you learn?): [Relearned how to open a file and strip unnecessary characters.]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 01/dylannakahodo/wordvalue.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    value = 0
+    for char in word:
+        try:
+            value += LETTER_SCORES[char.upper()]
+        except:
+            pass
```
Comment (pybites) [inline 01/dylannakahodo/wordvalue.py]: "Explicit is better than implicit" - better to use a specific Exception on except (not except/pass), see https://pybit.es/py-mistakes.html

Comment (dylannakahodo) [reply 01/dylannakahodo/wordvalue.py]: Thanks for reviewing! I realize this is a poor/lazy way of handling the key lookup in the dict after seeing other people's solutions.

Comment (pybites) [reply 01/dylannakahodo/wordvalue.py]: right: use dict.get which defaults to None or a default value you give it. 

I (Bob) ended up with this one-liner for calc_word_value:     

return sum(LETTER_SCORES.get(char.upper(), 0) for char in word)

Comment (pybites) [reply 01/dylannakahodo/wordvalue.py]: you can do this one here too now: https://codechalleng.es/bites/3/

Comment (dylannakahodo) [reply 01/dylannakahodo/wordvalue.py]: Cool thanks! I'll check it out.

**Comment by pybites** [comment]
cool

**Pull request merged**